conda_smithy/configure_feedstock.py

```diff
@@ -390,7 +390,7 @@ def main(forge_file_directory):
               'circle': {},
               'appveyor': {},
               'channels': {'sources': ['conda-forge'], 'targets': [['conda-forge', 'main']]},
-              'github': {'user_or_org': 'conda-forge', 'repo_name': ''},
+              'github': {'user_or_org': 'conda-forge', 'repo_name': '', 'branch_name': 'master'},
```

Comment:
Just wanted to let you know asap that unfortunately ```branch_name``` is not something I'd support here - it would result in redundancy of information, and would require a code change whenever we want a new branch. I would consider automatically determining this, but even then, there are caveats (like having a headless checkout) that we would have to consider carefully.
Are you able to back out the branch_name changes specifically @shadowwalkersb?